addons/website_sale/models/website.py

```diff
@@ -30,7 +30,10 @@ def _default_recovery_mail_template(self):
 
     enabled_portal_reorder_button = fields.Boolean(string="Re-order From Portal")
     salesperson_id = fields.Many2one(
-        string="Salesperson", comodel_name='res.users', domain="[('share', '=', False)]",
+        string="Salesperson",
+        comodel_name='res.users',
+        domain="[('share', '=', False)]",
+        default=lambda self: self.env.ref('base.user_admin').id,
```

Comment:
After talking about it with Jean, we believe it's safer to only assign it as default salesperson for the default website.
If users use multiple websites, they should be more aware of the configuration and manage it themselves ;).

```xml
        <record model="website" id="website.default_website">
            <field name="salesteam_id" ref="sales_team.salesteam_website_sales"/>
            <field name="salesperson_id" ref="base.user_admin"/>
        </record>
```